### ISSUE TITLE
Possible ChromeOS port?

### ISSUE THREAD

**Issue by xracecar** [milkytracker/MilkyTracker#88]
So the Chromebooks that are becoming more prevalent is going to my schools, which means I will be using a Chromebook a lot for school use. I would want a ChromeOS version of MilkyTracker since the new system would mean that I can't bring my Mac to school.

**Comment by alugarius** [comment]
isnt ChromeOS just linux?

**Comment by xracecar** [comment]
Yeah it runs off a linux kernel, but the school has like almost everything blocked besides the google play store. (where i got my gba emulator from)

**Comment by coderofsalvation** [comment]
Milkytracker runs fine on chromebooks using crouton or galliumos
